src/Compatibility/Microsoft.DotNet.PackageValidation/Resources.resx

```diff
@@ -118,42 +118,42 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CompatibleFrameworkInPackageValidatorHeader" xml:space="preserve">
-    <value>{0} assembly api surface area should be compatible with {1} assembly surface area so we can compile against {0} and run on {1} .framework.</value>
+    <value>Assembly API surface area of {0} should be compatible with assembly surface area of {1}. This ensures that it is possible to compile against {0} and then run on the {1} framework.</value>
```

Comment:
you will have to build and update the other resx files as well

Reply:
i went ahead and fixed the failures :)  just to save time so that we can port this to preview5

Reply:
Oh - thanks @Anipik - I had only done dotnet build on the two projects.

Reply:
I don't think it's necessary to get this into P5 unless you can merge it into your existing PR without resetting everything.

Reply:
> I don't think it's necessary to get this into P5 unless you can merge it into your existing PR without resetting everything.

i think we have a runtime change coming in so we can port this to preview 5 without reseting the builds.